lib/device/lib/logging.js

```diff
@@ -254,7 +260,9 @@ Logging.prototype.stop = function() {
 
   try {
     process.kill(-currentProcess.pid, 'SIGINT');
-  } catch (e) {}
+  } catch (e) {
+    //
```

Comment:
Yeah, it gets rid of this error:   263:15  warning  Empty block statement  no-empty